lib/http/base.js

```diff
@@ -246,19 +224,14 @@ function normalizeHeaders(headers)
     // browser-request will inject its own 'accept': 'application/json' header
     // and the browser XMLHttpRequest object will concatenate the two values and
     // send 'Accept': 'application/json, application/json' with the request
-    var headerNameLowerCase;
-    for (var headerName in headers)
-    {
-      if (headers.hasOwnProperty(headerName))
-      {
+    let headerNameLowerCase;
```

Comment:
reformatting this file should be in a separate PR - this keep alive feature is important enough to have it without all unimportant changes

Reply:
ok